src/engraving/libmscore/part.cpp

```diff
@@ -229,7 +229,7 @@ void Part::write(XmlWriter& xml) const
 
 int Part::nstaves() const
 {
-    return _staves.size();
+    return static_cast<int>(_staves.size());
```

Comment:
I believe, better change method signature to `size_t Part::nstaves() const`

Reply:
Same comment as above, many more changes needed in lots other places

Reply:
This one seems even worse, requiring more casting elsewhere (or even much bigger changes)

Reply:
Not worth the effoert IMHO

Reply:
not leading to cleaner code at a reasonable price

Reply:
I think it's better to do this later when most of the code will be changed to std